beeai/agents/triage_agent.py

```diff
@@ -106,6 +111,9 @@ def prompt(self) -> str:
           You are an agent tasked to analyze Jira issues for RHEL and identify the most efficient path to resolution,
           whether through a version rebase, a patch backport, or by requesting clarification when blocked.
 
+          **Important**: This agent focuses on bugs, CVEs, and technical defects that need code fixes.
```

Comment:
I think using imperative is better than third-person here (not to confuse the agent):

```suggestion
          **Important**: Focus on bugs, CVEs, and technical defects that need code fixes.
```